generate_and_send_a_creative_email_using_openai_and_stability_ai/main.bal

```diff
@@ -0,0 +1,84 @@
+import ballerina/io;
+import ballerina/mime;
+import ballerinax/googleapis.gmail;
+import ballerinax/openai.text;
+import ballerinax/stabilityai;
+
+configurable string gmailToken = ?;
+configurable string openAIToken = ?;
+configurable string stabilityAIKey = ?;
+
+final text:Client openAIText = check new ({auth: {token: openAIToken}});
+final gmail:Client gmail = check new ({auth: {token: gmailToken}});
+final stabilityai:Client stabilityAI = check new ({authorization: stabilityAIKey});
+
+public type EmailDetails record {|
+    string topic;
+    string recipientEmail;
+|};
+
+public function main(*EmailDetails emailDetails) returns error? {
+    fork {
+        // Generate greeting text and design in parallel
+        worker poemWorker returns string|error? {
+            text:CreateCompletionRequest textPrompt = {
+                prompt: string `Generate a creative poem on the topic ${emailDetails.topic}.`,
+                model: "text-davinci-003",
+                max_tokens: 1000
+            };
+            text:CreateCompletionResponse completionRes = check openAIText->/completions.post(textPrompt);
+            return completionRes.choices[0].text;
+        }
+
+        worker imageWorker returns byte[]|error? {
+            stabilityai:TextToImageRequestBody payload = {text_prompts: [{"text": emailDetails.topic, "weight": 1}]};
+            stabilityai:ImageRes listResult = check stabilityAI->/v1/generation/["stable-diffusion-v1"]/
```

Comment:
Why `stable-diffusion-v1` is added as path param while hard coding the value. If we don't change the value within the sample can't we simply use below?

```
stabilityAI->/v1/generation/stable\-diffusion\-v1/
                text\-to\-image.post(payload)
```

Reply:
Addressed in https://github.com/ballerina-guides/ai-samples/pull/67/commits/a252538fec1000750cfd86ec8ded9dee87a6047e.